### PR TITLE
Add support for Fuel TRANSFER receipts

### PR DESCRIPTION
**Pull request by DZakh**

### CODE REVIEW

File: scenarios/fuel_test/src/AllEventsHandlers.ts [hunk 1]
```diff
@@ -241,15 +241,15 @@ AllEvents.Transfer.handler(async ({ event }) => {
...
-  amount: SExtra.bigint,
-});
-AllEvents.Call.handler(
-  async ({ event }) => {
-    callSchema.assert(event.params)!;
-    expectType<AssertSchemaType<typeof event.params, typeof callSchema>>(true);
-  },
-  { wildcard: true }
-);
+// const callSchema = S.object({
```
Comment (JasoonS) [inline scenarios/fuel_test/src/AllEventsHandlers.ts:244]: Is there a reason you are leaving this here?

**Review by JonoPrest — APPROVED** [review]
Awesome looks good! Just the note on the commented code 👍🏼

File: codegenerator/cli/src/config_parsing/system_config.rs [hunk 2]
```diff
@@ -380,7 +378,8 @@ impl SystemConfig {
                         //there is no local_contract_config
                         if !contracts.get(&contract.name).is_some() {
                             Err(anyhow!(
-                                "Expected a local network config definition or a global definition"
+                                "Failed to parse contract '{}' for the network '{}'. If you use a global contract definition, please verify that the name reference is correct.",
+                                contract.name, network.id
```
[the remark this answers is not in the document]
Comment (JasoonS) [reply codegenerator/cli/src/config_parsing/system_config.rs:382]: Side Note: I think maybe give `make format` a run - these lines seem maybe a bit long

**Review by JasoonS — APPROVED** [review]
Looks clean to me

File: scenarios/fuel_test/src/AllEventsHandlers.ts (hunk 1 above)
Comment (DZakh) [reply scenarios/fuel_test/src/AllEventsHandlers.ts:244]: I wanted to keep it, so I can easily uncomment for future testing

**Pull request merged**